tasks/config.ts

```diff
@@ -131,117 +127,129 @@ export function load() {
       }
     });
   });
-}
-
-async function getPairsOraclesAndSymbols(hre: HardhatRuntimeEnvironment, deployer: SignerWithAddress) {
-  const factory = await getUniswapFactory(hre);
-  const pools = await getPools(hre);
-
-  const pairInfos: {
-    pair: { address: string; token0: string; token1: string; token0Symbol: string; token1Symbol: string };
-    pairOracle: { pairAddress: string; oracleAddress: string };
-    symbol: string;
-  }[] = [];
-  for (const poolPair of pools) {
-    let pairSymbol = getPoolKey(poolPair[0].token.address, poolPair[1].token.address, poolPair[0].name, poolPair[1].name);
-    const pairAddress = await factory.getPair(poolPair[0].token.address, poolPair[1].token.address);
-    const oracleAddress = (await hre.ethers.getContract(`UniswapPairOracle_${pairSymbol}`, deployer)).address;
-
-    if (hre.network.name.toLowerCase() == "rsk") {
-      pairSymbol = pairSymbol.replace("ETH", "RBTC").replace("WBTC", "ETHs");
-    }
 
-    pairInfos.push({
-      pair: {
-        address: pairAddress,
-        token0: poolPair[0].token.address,
-        token1: poolPair[1].token.address,
-        token0Symbol: poolPair[0].name,
-        token1Symbol: poolPair[1].name
-      },
-      pairOracle: { pairAddress: pairAddress, oracleAddress: oracleAddress },
-      symbol: pairSymbol
+  async function getPriceFeedsConfig(hre: HardhatRuntimeEnvironment, deployer: SignerWithAddress) {
+    const priceFeeds = Object.entries(PriceFeedContractNames).map(async ([key, value]) => {
+      const instance = await hre.ethers.getContract(value, deployer);
+      return { symbol: key, address: instance.address, decimals: instance.decimals ? await instance.decimals() : undefined };
```

Comment:
If we get undefined here for the decimals we should fail and not continue with `undefined`.
That could lead to unexpected behaviors.

Reply:
I see there is field feed of type AggregatorV3Interface which has decimals, but it is internal.

Reply:
We could implement getDecimals function like there is getLatestPrice()

Reply:
I see 2 contracts that inherit from `ICryptoPairOracle`:
1. BtcToEthOracleChinlink - which we haven't deployed yet.
2. WethToWethOracle - which we did deploy

We can expose add the `decimals` function (keep it standard instead of `getDecimals`) to the `ICryptoPairOracle` interface and add it to both contracts.
No real need to redeploy `WethToWethOracle` and `BtcToEthOracleChinlink` is not deployed yet and only exists on `mainnetFork`.

WDYT?

@AndrzejGisWideLab FYI

Reply:
Sounds good for me, do you want undefined checks in backend removed after that change?